ui/src/submissions/journals/components/__tests__/__snapshots__/JournalForm.test.jsx.snap

```diff
@@ -19,12 +18,11 @@ exports[`JournalForm renders 1`] = `
       <Field
         component={[Function]}
         label="* Journal Title"
-        name="journal_title"
+        name="journal_title.title"
```

Comment:
why we did this change?

Reply:
It's a leftover I missed during PR cleanup, I'll delete it